src/mb_enhanced_cover_art_uploads/providers/discogs.ts

```diff
@@ -109,17 +108,32 @@ export class DiscogsProvider extends CoverArtProvider {
         return metadata;
     }
 
+    static getFilenameFromUrl(url: URL): string | undefined {
+        // E.g. https://i.discogs.com/aRe2RbRXu0g4PvRjrPgQKb_YmFWO3Y0CYc098S8Q1go/rs:fit/g:sm/q:90/h:600/w:576/czM6Ly9kaXNjb2dz/LWltYWdlcy9SLTk4/OTI5MTItMTU3OTQ1/NjcwNy0yMzIwLmpw/ZWc.jpeg
```

Comment:
I'm curious, did you find any documentation about this URL pattern or did you just try to decode parts of the URL because it looked like Base64?

Reply:
I spotted parts ending in '=', which is a telltale sign of base64, so I messed around with trying to decode it and figured out the file name part. The signature was a case of unsuccessful trial-and-error to see if I could figure anything out about it, also using the old URL format, after which I looked at the docs on Thumbor (which Discogs used to use according to some IMU ticket) and found out the signature part in the old URLs. Then I looked up alternatives to Thumbor to compare with the new format, found imgproxy with the exact same URL structure, and then I called it quits since I highly doubt anyone will be able to reverse-engineer a maximisation strategy without resorting to things they shouldn't do (hacking/brute forcing). Brute forcing would be absolutely useless anyways because as soon as the key is found, they'll just change it.

I'm like 99% sure someone over at Discogs is keeping an eye on all of the threads which try to figure out how to get larger images, with a sort of evil laugh knowing that it's nigh impossible to do so. To that person: 🖕